pymc/distributions/distribution.py

```diff
@@ -594,7 +629,7 @@ def rv_op(
         random: Optional[Callable],
         moment: Optional[Callable],
         ndim_supp: int,
-        ndims_params: Optional[Sequence[int]],
+        ndims_params: Sequence[int],
```

Comment:
Why is this no longer `Optional` while everywhere else ndim becomes optional

Reply:
Users don't call this method, the user facing one `.dist` and `__new__` defines all these from the optional arguments